shared/actions/login/index.js

```diff
@@ -343,6 +350,21 @@ export function logoutDone () : AsyncAction {
   }
 }
 
+export function clearStoredSecret (username: string) : AsyncAction {
+  return (dispatch, getState) => {
+    const params: login_clearStoredSecret_rpc = {
+      ...makeWaitingHandler(dispatch),
```

Comment:
Do we want this?

This basically keeps track if the login stuff is doing anything to disable buttons and show a waiting animation to the user.

Not sure it makes sense to show the user a waiting animation for this, but maybe I'm wrong